features/api_token_info.feature

```diff
@@ -0,0 +1,151 @@
+Feature: api.u.pro.token_info.v1
+
+  @uses.config.contract_token
+  Scenario Outline: Get token info from api
+    Given a `<release>` `<machine_type>` machine with ubuntu-advantage-tools installed
+    When I run `pro api u.pro.token_info.v1 --args token=$behave_var{contract_token}` with sudo
+    Then API data field output matches regexp:
+      """
+      {
+        "attributes": {
+          "account": {
+            "id": ".*",
+            "name": "UA Client Test"
+          },
+          "contract": {
+            "id": ".*",
+            "name": "server-team-ua-client-ci-uaa"
+          },
+          "services": [
+            {
+              "auto_enabled": true,
+              "available": false,
+              "description": "Scalable Android in the cloud",
+              "entitled": true,
+              "name": "anbox-cloud"
+            },
+            {
+              "auto_enabled": false,
+              "available": true,
+              "description": "Common Criteria EAL2 Provisioning Packages",
+              "entitled": true,
+              "name": "cc-eal"
+            },
+            {
+              "auto_enabled": false,
+              "available": true,
+              "description": "Security compliance and audit tools",
+              "entitled": true,
+              "name": "cis"
+            },
+            {
+              "auto_enabled": true,
+              "available": true,
+              "description": "Expanded Security Maintenance for Applications",
+              "entitled": true,
+              "name": "esm-apps"
+            },
+            {
+              "auto_enabled": true,
+              "available": true,
+              "description": "Expanded Security Maintenance for Infrastructure",
+              "entitled": true,
+              "name": "esm-infra"
+            },
+            {
+              "auto_enabled": false,
+              "available": true,
+              "description": "NIST-certified FIPS crypto packages",
+              "entitled": true,
+              "name": "fips"
+            },
+            {
+              "auto_enabled": false,
+              "available": false,
+              "description": "Preview of FIPS crypto packages undergoing certification with NIST",
+              "entitled": true,
+              "name": "fips-preview"
```

Comment:
[similar to this one](https://github.com/canonical/ubuntu-pro-client/pull/3262/files#r1724908133) - availability varies between releases.